### ISSUE TITLE
Development Mura changes are reflecting in Staging and Production as well. How to stop reflcting in staging and Production and create separate deployments?

### ISSUE THREAD

**Issue by jeevanpulugu** [blueriver/MuraCMS#2198]
I am new to Mura. There is a website already built and from Development admin panel if I do some changes then those changes are getting reflected in Staging and Production as well.

We are using Mura 6.2.6205

How can I stop this behavior and make separate deployments for Staging and Production?

Thanks

**Comment by rwatts3** [comment]
I would take a look at your DB settings for each environment. 
Sometimes it's easy to overlook the db settings and they may be pointed to the same datasource.
If they are then, you'll notice updates rolling into production etc.

**Comment by rwatts3** [comment]
Sorry, more specifically check your settings.ini.cfm file.

**Comment by jeevanpulugu** [comment]
Hi Ryan,

Thanks for responding on this post.

I have gone through settings.ini.cfm file, it is ponting to datasource=Mura_DB_New

And in CF Admin this datasource is pointing to Production database in all the 3 environments. So what I have to done I have duplicated Production Database to Development and Staging. Then from CF Admin I have pointed Mura_DB_New data source to duplicated database.

Now it is working fine for displaying the site in browser. But I got stuck with another problem.

When I am trying to publish the new content it is throwing me 500 Internal Server Error. I tried to debug this by making debug on from CF Admin but I didn't get any information.

Can you help me on this ?

Thanks,
Jeevan Pulugu

**Comment by rwatts3** [comment]
Ah I see, 

Ok you could try two things here.
1. Try to reload the application and see if it is not picking up the new datasource updates.
2. If that doesn't work if you're using lucee, coldfusion, or railo, try to clear your component cache and see if that works.

**Comment by rwatts3** [comment]
Also if you're using CF, you should be able to see log .txt files in your Coldfusion Directory, I do not remember the path off the top of my head. That file should tell you the most recent errors logged from the server.

**Comment by jeevanpulugu** [comment]
I have restarted my services and cleared component cache as well. But that didn't help.

When I verified my logs I have seen the errors like below image. I think some connections might be wrong in Database. As I have duplicated the initially created database something it is not able to find.

![image](https://cloud.githubusercontent.com/assets/5284754/14752637/1550dd1a-08ee-11e6-986a-12eaf6552c6a.png)
